.travis.yml

```diff
@@ -7,7 +7,7 @@ python:
   - "3.4"
   - "3.5"
 sudo: false
-script: nosetests test --verbose
+script: nosetests test --verbose --processes=4 --process-timeout=540
```

Comment:
You can pass `--processes=-1` ([docs](https://nose.readthedocs.org/en/latest/man.html#cmdoption--processes)) and it will automatically select  the number based on the number of cores. Maybe that's better?